### PR TITLE
Members Only Rails App

### PR DESCRIPTION
**Pull request by Yors-git**
In this app I worked on an app that shows posts by title and its corresponding body, but it only shows the author of each post if the user is logged in.
- Built with Ruby on Rails
- Used Devise Gem
- Set up restrictions to controller actions

### CODE REVIEW

File: app/views/posts/index.html.erb
Comment (teekaytech) [inline app/views/posts/index.html.erb]: These are logic that should be encapsulated in a helper. Check my comments for details.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made laudable progress in this milestone.
However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] The application should have Sign in/Sign out links present in any view of the app ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_in_out_link_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49722686/))
- This should be achieved using the `application.html.erb` application layout file, so they are not manually repeated in every view.

- [x] As indicated in the inline comments below, there should be absolutely NO logic in views. Check [this](http://codefol.io/posts/Where-Do-I-Put-My-Code) and [this](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) resources on how to use helpers to encapsulate view-related logic.



**[ OPTIONAL CHANGES ]**

- [x] You can improve on the UI of the app. This will make your app more attractive to see.

* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by Dandush03 — APPROVED** [review]
# Status: Project Approved :rocket: 
## Keep Up The Good Work :muscle: 
# Happy Coding :muscle: :computer:

**Pull request merged**
